R/utils.R

```diff
@@ -51,6 +51,9 @@ strrep <- function(x, times) {
   }, list(x = x, times = times), MoreArgs = list()), use.names = FALSE)
 }
 
+# Backport for R < 4.0
+deparse1 <- function(expr, ...) paste(deparse(expr, ...), collapse = "\n")
```

Comment:
I converted this to a newline because I think it looks a little better (see snapshot test below)

Reply:
One reason I might prefer `" "` is IINM the `deparse` rules for wrapping are not strictly tied to the terminal `width`; using `" "` means "let the terminal decide when to wrap".

Not particularly important in any case, really this fix is about not building the string incorrectly.

Reply:
Interesting — I think we generally assume that the terminal always wraps badly so we take control as much as possible. But yeah, not really important here since this is just about avoiding an errorful edge case.

Reply:
I think that's also not wrong, it's a matter of whether `deparse` or terminal has better wrapping rules... more control would come from `strwrap` but that's usually too much effort IME. I also don't remember having good experience trying to get `width.cutoff` to work naturally, but I could be wrong.

Reply:
Plus in this case the first ("reason") and last ("is not TRUE") lines are longer than the others, which you'd also like to take into account somehow, but ideally you want to use code wrapping rather than prose wrapping, so doing it 100% correctly seems like a very hard problem.

Reply:
:100: